sdk/metric/sdk.go

```diff
@@ -152,6 +152,8 @@ var (
 	_ api.AsyncImpl     = &asyncInstrument{}
 	_ api.SyncImpl      = &syncInstrument{}
 	_ api.BoundSyncImpl = &record{}
+
+	ErrUninitializedInstrument = fmt.Errorf("Use of an uninitialized instrument")
```

Comment:
nit, keeping everything lower case eases subsequent wrapping:

```suggestion
	ErrUninitializedInstrument = fmt.Errorf("use of an uninitialized instrument")
```

E.g.

```
fmt.Errorf("collection error: %w")
```

becomes `error("collection error: use of an uninitialized instrument")` instead of  `error("collection error: Use of an uninitialized instrument")`.

Reply:
Done.